### PR TITLE
Update csv-io example with metadata files

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: examples/csv-io/csv_io.bal [hunk 1]
```diff
@@ -81,17 +80,13 @@ public function main() returns @tainted error? {
...
     Employee[] employees = [];
     employees[0] = {id: "1", name: "Allen", salary: 300000.0};
     employees[1] = {id: "2", name: "Wallace", salary: 200000.0};
     employees[2] = {id: "3", name: "Sheldon", salary: 1000000.0};
     foreach var employee in employees {
         var result = employeeTable.add(employee);
-        if (result is error) {
-            log:printError("Error occurred while adding data to table: ",
-                            err = result);
-        }
```
Comment (daneshk) [inline examples/csv-io/csv_io.bal:94]: Don't we get any error while adding to the table

Comment (MadhukaHarith92) [reply examples/csv-io/csv_io.bal:94]: AFAIK, with the new table syntax, the `add` method doesn't return an error and instead, it panics. See https://github.com/ballerina-platform/ballerina-lang/blob/master/langlib/lang.table/src/main/ballerina/src/lang.table/table.bal#L83

Comment (daneshk) [reply examples/csv-io/csv_io.bal:94]: ok, then can we mention actual return type instead of var. is it () here?

Comment (daneshk) [reply examples/csv-io/csv_io.bal:94]: it does not return anything. we can just remove assignment

Comment (MadhukaHarith92) [reply examples/csv-io/csv_io.bal:94]: Ack. Updated in a63e083

File: examples/csv-io/csv_io.metatags [hunk 2]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to read/write from/to a CSV file using the CSV channel.
```
Comment (Samallama) [inline examples/csv-io/csv_io.metatags:1]:
```suggestion
description: BBE on how to read/write from/to a CSV file using the CSV channel in Ballerina.
```

**Pull request merged**